xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/viewers/comments.js

```diff
@@ -65,69 +64,6 @@ viewers.Comments = Class.create({
       item._x_number = elementId.substring(elementId.lastIndexOf("_") + 1) - 0;
     });
   },
-  /**
-   * Ajax comment deletion.
-   * For all delete buttons, listen to "click", and make ajax request to remove the comment. Remove the corresponding
-   * HTML element on succes (replace it with a small notification message). Display error message (alert) on failure.
-   */
-  addDeleteListener : function() {
-    $$(this.xcommentSelector).each(function(item) {
-      // Prototype bug in Opera: $$(".comment a.delete") returns only the first result.
-      // Quick fix until Prototype 1.6.1 is integrated.
-      item = item.down('a.delete');
-      if (!item) {
-        return;
-      }
-      item.observe('click', function(event) {
-        item.blur();
-        event.stop();
-        if (item.disabled) {
-          // Do nothing if the button was already clicked and it's waiting for a response from the server.
-          return;
-        } else {
-          new XWiki.widgets.ConfirmedAjaxRequest(
-            /* Ajax request URL */
-            item.readAttribute('href') + (Prototype.Browser.Opera ? "" : "&ajax=1"),
-            /* Ajax request parameters */
-            {
-              onCreate : function() {
-                // Disable the button, to avoid a cascade of clicks from impatient users
-                item.disabled = true;
-              },
-              onSuccess : function() {
-                // Remove the corresponding HTML element from the UI and update the comment count
-                var comment = item.up(this.xcommentSelector);
-                // If the form is inside this comment's reply thread, move it back to the bottom.
-                if (this.form && this.form.descendantOf(comment.next('.commentthread'))) {
-                  this.resetForm();
-                }
-                // Replace the comment with a "deleted comment" placeholder
-                comment.replace(this.createNotification("$services.localization.render('core.viewers.comments.commentDeleted')"));
-                this.updateCount();
-                // fire an event for the annotations to know when a comment / annotation is deleted
-                // FIXME: This is not the best way to go because the Annotations system should be in charge of properly deleting annotations,
-                // not the Comments system. Try to find an alternative for the future.
-                if (comment.hasClassName('annotation')) {
-                  document.fire("xwiki:annotation:tab:deleted");
```

Comment:
You don't seem to fire this event anymore. I know the comment says it should be fixed but there may be code that expects this event.